### DISCUSSION TITLE
SQLite share in cloud

### DISCUSSION THREAD

**Discussion by ruham20** [sqlitebrowser/sqlitebrowser#3140] [General]
Can anyone help me to share a db in cloud?

**Comment by chrisjlocke** [comment]
Have you tried dbhub.io?
Create an account there, create a certificate, then in DB4S, you can import that certificate.  From within DB4S you can then upload revisions of the database.

Can go into more details and a walkthrough if you need one. :)

**Comment by chrisjlocke** [comment]
If you do sign up and get a wacky error (because you're using Firefox) then slap Firefox.

![image](https://user-images.githubusercontent.com/7872218/185260461-627d6ae8-f007-4b84-9638-51f2dc2949ab.png)

It's a bit more strict when it comes to cookies.

![image](https://user-images.githubusercontent.com/7872218/185260587-ba3ce4b1-26a5-4e97-b2e1-3c588b2ee620.png)

Clicl the shield thingy, then toggle the toggle thingy.

You should then be able to login.

![image](https://user-images.githubusercontent.com/7872218/185260674-6cb290f5-946f-45ac-842d-679ce1d59eb2.png)

Once you have logged in, the thing I always forget is setting a username.  While this can be the same as the signup username, it is a different username (if that makes sense ... which it probably doesnt).  The previous username was for signing up for auth0.

**Reply by ruham20** [reply]
Thank you.
But how it can synchronize ?

**Reply by chrisjlocke** [reply]
You simply push up 'commits', same as you would do for Git.
When you want to work on the database, download it, work on it, then upload it.

Someone else can download it and upload their version.
